feature/home/src/main/java/io/filmtime/feature/home/VideoThumbnailCard.kt

```diff
@@ -16,11 +30,30 @@ fun VideoThumbnailCard(
   videoThumbnail: VideoThumbnail,
   onClick: () -> Unit,
 ) {
+  var selected by remember { mutableStateOf(false) }
+  val scale by animateFloatAsState(if (selected) 1.1f else 1f, label = "scale-transition")
```

Comment:
After adding scale animation on focus, which is reasonable only on TV, we some size issues on phone. The thumbnail is scaled after navigating back from detail screen on phone

<img src="https://github.com/moallemi/Film-Time/assets/1250691/91cce438-8d0b-4555-bc56-fbcaa5e575b0" width=50% />